### DISCUSSION TITLE
Project contribution

### DISCUSSION THREAD

**Discussion by RomainTiennot** [pfsensible/core#127] [General]
Hello @opoplawski ,
I'd like to contribute to the project to improve some of the modules, but I'm not sure where to start.
I'd like to know if you have time to check the MRs and make changes if necessary?
Because I'd like to contribute, but if they remain in approval or on hold because there are too many things that don't fit and I don't get any feedback, that would be a shame.
I can see that you've made some changes, but do you have time to devote to novice contributors?

Have a nice day!

**Comment by opoplawski** [comment]
Hi @RomainTiennot ,
I don't have as much time as I would like, but I'll do my best to help you out.  You may want to start looking at the `module_base_extended` branch as I've been working on some changes there and a tool to help with building modules that I am hoping will help others create or even update modules more easily.  What area are you most interested in?

**Comment by RomainTiennot** [comment]
Hi @opoplawski,
Thanks for your reply.
Thanks, I'll take a look at the branch. I'm interested in updating modules and creating new ones. I would like to be able to configure antierement my pfSense via Ansible and I miss things.
I'm starting from afar. Do you have any advice/links to help me continue development?
